pkg/kubevip/config_generator.go

```diff
@@ -53,6 +53,17 @@ const (
 	//vipPacket defines which project within Packet to use
 	vipPacketProject = "vip_packetproject"
 
+	//bgpEnable defines if BGP should be enabled
+	bgpEnable = "bgp_enable"
+	//bgpRouterID defines the routerID for the BGP server
+	bgpRouterID = "bgp_routerID"
```

Comment:
nitpic: This doesn't seem to fit the naming pattern (`lower_loweralowerb`)

Reply:
Fixed this, in the lates commit. 👍